src/Keboola/Console/Command/AddFeaturesFromCSV.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Keboola\Console\Command;
+
+use Keboola\Csv\CsvFile;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class AddFeaturesFromCSV extends AddFeatures
+{
+    public const ARG_CSV_FILE = 'csvFile';
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('manage:add-feature-to-templates-from-csv')
```

Comment:
nebylo by lepší `manage:add-feature-to-templates:from-csv`

Reply:
nebyl jsem si jistej s tou naming convention, ale toto vypada dobre